cakephpsphinx/themes/cakephp/layout.html

```diff
@@ -49,7 +49,7 @@
         {%- endif -%}
     {% endfor %}
 
-    <link rel="search" type="application/opensearchdescription+xml" title="Search within {{ docstitle }}" href="{{ pathto('_static/opensearchdescription-book-3-x.xml', 1) }}">
+    <link rel="search" type="application/opensearchdescription+xml" title="Search within {{ docstitle }}" href="{{ pathto('_static/opensearchdescription-book-4-x.xml', 1) }}">
```

Comment:
Wouldn't this cause the 3.x docs to break instead? Could we interpolate the `release` template variable instead of having a hardcoded version?

Reply:
Good point. Would there then be a ../opensearchdescription-book-X-x.xml for each version? I didn't look into how the sphinxtheme version matches with the docs version.